### PR TITLE
Up date

### PR DESCRIPTION
**Pull request by mildlyincompetent**
wow

### CODE REVIEW

File: display/projectiles.py [hunk 1]
```diff
@@ -6,21 +6,27 @@
 
 class Projectile(pygame.sprite.Sprite):
 
-    def __init__(self, icon, velocity, oscillate_height = 0, oscillate_rate = 1):
+    pixel_scale = 5
```
Comment (trickeydan) [inline display/projectiles.py]: Repetition of values is unnecessary

File: display/projectiles.py [hunk 2]
```diff
@@ -29,10 +35,13 @@ def __init__(self, icon, velocity, oscillate_height = 0, oscillate_rate = 1):
 
     def draw_image(self):
         ifile = pygame.image.load(os.path.join("assets", "items", self.icon + ".png"))
-        self.image = pygame.transform.scale(ifile, (150, 150))
+        self.image = pygame.transform.scale(ifile, (15*Projectile.pixel_scale, 15*Projectile.pixel_scale))
```
Comment (trickeydan) [inline display/projectiles.py]: Please could you clarify where `15` comes from here?

File: display/projectiles.py [hunk 3]
```diff
@@ -29,10 +35,13 @@ def __init__(self, icon, velocity, oscillate_height = 0, oscillate_rate = 1):
...
 
     def update(self):
         self.rect.x += self.velocity
 
         if self.oscillate_height != 0:
             self.rect.y = self.default_y + math.floor(math.sin((self.rect.x * self.oscillate_rate / screen_width) * 2 * math.pi) * self.oscillate_height)
+
+        if self.rect.x < -500 or self.rect.x > screen_width+500:
```
Comment (trickeydan) [inline display/projectiles.py]: Ew, lack of spaces :nauseated_face:

File: display/projectiles.py (hunk 1 above)
Comment (mildlyincompetent) [reply display/projectiles.py]: No.

File: display/projectiles.py (hunk 2 above)
Comment (mildlyincompetent) [reply display/projectiles.py]: Ye the size of the image yo.

File: display/projectiles.py (hunk 3 above)
Comment (mildlyincompetent) [reply display/projectiles.py]: here:                    have some more.

**Comment by trickeydan** [comment]
Pineapples.txt approves of this change

**Pull request merged**